### DISCUSSION TITLE
Tutorial Visualisasi Data dengan Peta Jabodetabek

### DISCUSSION THREAD

**Discussion by AndiSofyan01** [teamalgoritma/community#888] [Q&A] [answered]
Hi Team,

Saya ingin visualisasi data hasil akuisisi daerah Jabodetabek ke dalam peta seperti contoh heatmap di tableau atau di lookers. Namun, ketika saya click petanya tidak ter reflect untuk peta spesifik jabodetabek. Mohon bantuannya untuk step by step nya. Terima kasih.

**Comment by Yusraf002** [comment]
Dear @AndiSofyan01 

Terimakasih sudah bertanya melalui forum ini, mengenai pertanyaan ibu diatas kiranya boleh saya konfirmasi terlebih dahulu kiranya visualisasi yang akan dibuat oleh ibu ini untuk penggunaan bahasa pemrograman apa / aplikasi apa ?

Best Regards,
Team Algoritma

**Reply by AndiSofyan01** [reply]
Hi Pak,

Saya ingin membuat visualisasi nya menggunakan Tableau

**Reply by Yusraf002** [reply, marked as answer]
Untuk heatmap, hanya bisa sampai tingkat provinsi. Tidak bisa tingkat kota. Kalau mau kota jadinya point2 titik kota tersebut sehingga tidak reflect seperti heatmap.
Alternatifnya, tetap gunakan kota. Namun, besarkan size dari point (circle) tersebut.

**Reply by IrfanChairurrachman** [reply]
Halo @AndiSofyan01 , untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.
